Composer/packages/server/src/locales/en-US.json

```diff
@@ -35,12 +35,18 @@
   "a_property_is_a_piece_of_information_that_your_bot_eccd34bf": {
     "message": "A property is a piece of information that your bot will collect. The property name is the name used in Composer; it is not necessarily the same text that will appear in your bot''s messages."
```

Comment:
Revert this file back to the version on feature/bot-projects. This PR need not update the locale file. We have an automated build process to update the locales files